docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -101,27 +101,11 @@ to `true` on an API instance. This will prevent serving data on API requests fro
 
 ## Partitioning
 
-[Partitioning][wiki-partitioning] is an extremely effective optimization for
-improving data access. It effectively "shards" the data between multiple tables,
-splitting them by a defined attribute. An incoming query would be checked for
-this attribute, and **only** valid partitions required to satisfy it are
-selected. This results in faster refresh times due to unnecessary data not being
-scanned and processed, and possibly even reduced cost, depending on your
-database solution. Cube.js supports partitioning data using the `timeDimension`
-property in [a pre-aggregation definition][ref-schema-ref-preaggs].
-
-### <--{"id" : "Partitioning"}-->  Time partitioning
-
-Time-based partitioning is especially helpful for incremental refreshes; when
-configured, Cube.js will only refresh partitions as necessary. Without
-incremental refreshing, Cube.js will re-calculate the entire pre-aggregation
-whenever [the refresh key][ref-schema-ref-preaggs-refresh-key] changes.
-
-<!-- prettier-ignore-start -->
-[[warning |]]
-| Partitioned rollups currently cannot be used by queries without time
-| dimensions.
-<!-- prettier-ignore-end -->
+[Partitioning][wiki-partitioning] is an extremely effective optimization
```

Comment:
how about if we say "an effective optimization tool" or "an effective optimization technique"

Reply:
Just a minor comment. I'm also OK with leaving the sentence as is 😄